lib/travis/build/git/clone.rb

```diff
@@ -40,6 +40,7 @@ def checkout
           def clone_args
             args = "--depth=#{depth}"
             args << " --branch=#{branch}" unless data.ref
+            args << "--quiet" if quiet?
```

Comment:
you need to add a space before the `--`, also, you don't need to keep closing and opening new PRs :)